public/styles/index.scss

```diff
@@ -33,15 +33,13 @@
 
 // @primer/css imports
 @import "@primer/css/support/index.scss";
-
 @import "@primer/css/base/normalize.scss";
 @import "@primer/css/base/kbd.scss";
 @import "@primer/css/buttons/index.scss";
 @import "@primer/css/marketing/index.scss";
 @import "@primer/css/tooltips/index.scss";
 @import "@primer/css/utilities/padding.scss";
 @import "@primer/css/pagination/index.scss";
-
 @import "base";
```

Comment:
We really need space here, this is a divider between primer and our styles.

Reply:
Added a space here too to clarify